aequilibrae/project/network/network.py

```diff
@@ -52,7 +54,17 @@ def skimmable_fields(self):
                     break
 
         all_fields.append('distance')
-        return all_fields
+        real_fields = []
```

Comment:
I'm not entirely sure what is happening here, do you mind reminding me of the data structure?

Reply:
What this does is to return all fields that can be skimmed. Default fields (distance excepted) should not be considered skimmable, so what we are doing here is to get all numeric fields except the required fields, plus distance

Reply:
right, thanks